Tests/SonarScanner.MSBuild.PreProcessor.Tests/SonarWebServiceTest.cs

```diff
@@ -83,15 +83,15 @@ public void Ctor_NullLogger_Throws()
         [TestMethod]
         public void LogWSOnError()
         {
-            this.downloader.Pages["http://myhost:222/api/qualityprofiles/search?projectKey=foo+bar"] = "trash";
+            this.downloader.Pages["http://myhost:222/api/qualityprofiles/search?project=foo+bar"] = "trash";
```

Comment:
are these tests on a SQ instance? can you add tests with the latest SQ and the LTS SQ?

Reply:
No, these are unit test, agnostic from SQ version.

Reply:
Are there any integration tests with SQ?
If there are none, it means it would rely on sonar-dotnet ITs for detecting problems, which is under optimal (would happen after a new release)

Could you add ITs with SQ in this sprint, to ensure quick feedback ?